app/views/layouts/application.html.erb

```diff
@@ -29,6 +29,7 @@
       <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
That sounds good to me; thanks. I just wanted to merge this today -- I probably won't publish this until next week, so we won't deploy with this version and I can merge in a new version after an additional tweak. Thanks.